include/drivers/i2c/sun6i-a31-i2c.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef DRIVERS_I2C_SUN6I_A31_I2C_H
+#define DRIVERS_I2C_SUN6I_A31_I2C_H
```

Comment:
why is this "DRIVERS_I2C_SUN6I_A31_I2C_H" instead of "DRIVERS_SUN6I_A31_I2C_H" to mach what it is called elsewhere?

Reply:
> The header guard should be the same as the path you would need to put in your `#include` directive, but with the following changes:
> - All letters should be uppercase.
> - All hyphens, slashes, and periods should be replaced by underscores.

So because `i2c` is part of the path, it should be in the header guard